frontend/src/components/commons/SideNavigation.jsx

```diff
@@ -37,16 +37,29 @@ class SideNavigation extends Component {
   };
 
   state = {
-    currentNode: this.props.currentNode,
-    currentBody: this.props.navSpecs[this.props.currentNode].body
+    currentNode: this.props.currentNode
   };
 
   changeNode(id) {
-    this.setState({ currentNode: id, currentBody: this.props.navSpecs[id].body });
+    this.setState({ currentNode: id });
+    this.refs[id].scrollIntoView({ block: "start", behavior: "smooth" });
   }
 
+  onScroll = () => {
+    const curVw = document.getElementById("side-nav-grid-content-cell").getBoundingClientRect();
+    var id = 0;
+    for (var i = this.props.navSpecs.length - 1; i >= 0; i--) {
+      var tab = this.props.navSpecs[i];
+      var elmnt = document.getElementById(tab.text).getBoundingClientRect();
+      // Verify that the top is at the same height or higher than tht etop and the bottom is still below the top
```

Comment:
Minor observation here: _"...or higher than **tht etop** and the bottom..."_

Reply:
fixed in 5d3f4f3